.gitattributes

```diff
@@ -1,5 +1,8 @@
 # Auto detect text files and perform LF normalization
-* text=auto
+# We explicitly set eol to LF on checkin and checkout
+*.py text=set eol=lf
+*.rst text=set eol=lf
+*.inc text=set eol=lf
```

Comment:
Why specific extensions? What would be the drawback of wild-carding this?
For example, we're missing markdown.

Reply:
Because I don't want to force the conversion on any binary files (like the zip files that will be added in a different PR)

Added many more text types.